### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by zookeeper77** [glutanimate/popup-dictionary#18]
#### Problem description

I downloaded the add on, nothing pops up when I double click or when I press Cntrl+Shift+D. I have a lot of other add-ons, so I don't know which ones are conflicting!

#### Checklist

*Please replace the space inside the brackets with an **x** if the following items apply:*

 - [x ] I've restarted Anki to see if it helps
 - [x] I've verified that I use the latest version of the add-on by redownloading it from AnkiWeb
 - [x] I've verified that I use the latest version of Anki by checking at https://apps.ankiweb.net#download
 - [ ] I've tried to disable other add-ons to see if there are any interactions present
 - [ ] My issue disappears when I hold shift while starting Anki.
 - [ ] I've checked if anyone else reported this problem before by looking through the issue reports. I also checked to see if there is a section about known issues in the add-on description, documentation, or README.
 

#### Information about your Anki set-up

*Please fill out the section corresponding with your Anki version:*

##### If you are using Anki 2.1

*Please open Anki, go to Help → About, click on "Copy Debug Info", and paste the result between the backticks below (if the button does not appear you are using an older version of Anki 2.1 and will need to update first):*

```
Anki 2.1.14 (7b93e985) Python 3.6.7 Qt 5.12.1 PyQt 5.11.3
Platform: Mac 10.14.5
Flags: frz=True ao=True sv=1

Add-ons:

Edit Field During Review
Speed Focus Mode auto-alert auto-reveal auto-fail
Special Fields
put ALL due learning cards first fork for 21
HoochiePapa Randomize New Queue
Image Occlusion Enhanced for Anki 21 alpha
load balancer
HoochieMama Randomize Rev Queue
Symbols
Night Mode
Pop-up Dictionary beta
More Decks Stats and Time Left
HoochieBaby Queue Controller  Day-Lrn Q
Fastbar another toolbar for Anki 21 beta
Export cards selected in the Browser
Customize Sidebar
Progress Bar
More Overview Stats
Highlight Search Results in the Browser
Customize Keyboard Shortcuts
Quick Colour Changing
Button Colours Good Again
Batch Editing
Mini Format Pack
Add Hyperlink
ReMemorize Rescheduler with sibling and logging v114
Clickable Tags v10
Frozen Fields
add-on window search bar
Syllabus - Anki Statistics and Export by Tag and Deck
Echelon - Improved Hierarchical Tags for Anki 21
Create Filtered Deck from the Browser
Web Browser - Search selection Import texts and images automatically
Advanced Browser
Review Heatmap
```

#####  If you are using Anki 2.0

*Please open Anki, go to Help → About, find the two lines starting with `Version:` and `Qt`, and copy-paste them between the backticks below:*

```

```

*Please fill in details about your operating system (Windows/macOS/Linux, which version):*

- OS: MacOS
- version: Mojave

*Please open Anki, go to Tools → Add-ons, take a screenshot of your installed add-ons, and paste it below:*
<img width="449" alt="Screen Shot 2019-08-07 at 11 01 55 AM" src="https://user-images.githubusercontent.com/53875285/62646268-cc4a3800-b902-11e9-8f28-e4a8127e4abc.png">

#### Error message (if any)

*If you've received an error message, please copy and paste it between the backticks below:*

```python

```

**Comment by BlueGreenMagick** [comment]
Try disabling Night Mode

**Comment by glutanimate** [comment]
Right, as BlueGreenMagick mentioned, Night Mode is usually the culprit with add-on conflicts that affect the reviewer. But if that doesn't work, I would try to systematically walk through your add-ons to see if you can identify the culprit. You should probably start by disabling all add-ons except Pop-up Dictionary in order to confirm that this is an add-on conflict in the first place.
